src/extension.ts

```diff
@@ -841,22 +893,18 @@ function migrateStringSettingToArray(id: string): void {
     .inspect<string | string[]>(id)!;
 
   if (typeof setting.globalValue === "string") {
-    workspace
-      .getConfiguration("metals")
-      .update(
-        id,
-        setting.globalValue.split(" ").filter(e => e.length > 0),
-        ConfigurationTarget.Global
-      );
+    workspace.getConfiguration("metals").update(
```

Comment:
Is this formatting change caused by a Prettier version mis-match? @gabro is there a way to enforce a stable version of Prettier in CI?

Reply:
Yes, it's sufficient to `yarn add --dev prettier` and then run `yarn prettier --check` on the CI.

Reply:
Added an issue for this.